parsl/data_provider/data_manager.py

```diff
@@ -47,13 +48,18 @@ class DataManager(ParslExecutor):
     to it, and DataFutures are returned.
     """
 
-    default_data_manager = None
-
     @classmethod
     def get_data_manager(cls, max_threads=None, executors=None):
-        if cls.default_data_manager is None:
-            cls.default_data_manager = DataManager(max_threads=max_threads, executors=executors)
-        return cls.default_data_manager
+        """Return the DataManager of the currently loaded DataFlowKernel.
+
+        Note that an error will be raised if one tries to create a File before
```

Comment:
this comment maybe belongs on `File`, not here.

It is (should be?) possible to create a file attached to an arbitrary DFK, not just the current globally loaded one, and this phrasing doesn't reflect that.

Reply:
agreed, i'll move it and clarify